bin/sozo/src/commands/inspect.rs

```diff
@@ -135,6 +135,17 @@ struct EventInspect {
     selector: String,
 }
 
+#[derive(Debug, Serialize)]
+struct ExternalContractInspect {
+    contract_name: String,
+    instance_name: String,
+    class_hash: String,
+    status: String,
+    salt: String,
+    constructor_calldata: Vec<String>,
+    address: String,
+}
```

Comment:
```suggestion
#[derive(Debug, Tabled, Serialize)]
struct ExternalContractInspect {
    #[tabled(rename = "External Contract")]
    contract_name: String,
    #[tabled(rename = "Instance Name")]
    instance_name: String,
    #[tabled(skip)]
    class_hash: String,
    #[tabled(rename = "Status")]
    status: ResourceStatus,
    #[tabled(skip)]
    salt: String,
    #[tabled(skip)]
    constructor_calldata: Vec<String>,
    #[tabled(rename = "Contrat Address")]
    address: String,
}
```
wdyt about something like this as default printing?
<img width="827" alt="image" src="https://github.com/user-attachments/assets/a172f636-5edc-428c-a944-885a042bf9ad" />